### DISCUSSION TITLE
Uncaught TypeError: at.settings.fillFromJson is not a function

### DISCUSSION THREAD

**Discussion by goodgame365** [CoderLine/alphaTab#673] [Q&A] [answered]
After up to 1.3.0 
It used to work normally, see this url :https://github.com/CoderLine/alphaTab/issues/285 

but now it's wrong
now fillFromJson is erro It's :

Uncaught TypeError: at.settings.fillFromJson is not a function

why

**Comment by Danielku15** [comment, marked as answer]
Indeed the JSON serialization system changed, but this change was already introduced in 1.2.0. It was an unintended breaking change and I will try to provide a backwards compatible version soon (1.2.2 and 1.3.0-alpha.###)

I entered https://github.com/CoderLine/alphaTab/issues/674 to track this misbehavior.

**Reply by Danielku15** [reply]
New alphaTab builds are available and API compatibility should be restored.

**Reply by goodgame365** [reply]
Thank you very much. It's OK indeed. The boss is too harmful

**Comment by goodgame365** [comment]
However, I'm very sorry to find new problems. Chrome browser often crashes

Your website（ https://www.alphatab.net/ ）Such a mistake also occurred

The error message displayed is as follows:

Error code: Status_ ACCESS_ VIOLATION

**Reply by Danielku15** [reply]
I don't thing that this has something to do with alphaTab but is an issue on your PC. This status code means Chrome is fully crashing unexpectedly, this is a bug in Chrome itself, may or maybe not caused by some things alphaTab is doing.
 
Follow the usual Chrome recommendations on the STATUS_ACCESS_VIOLATION code: update to the latest version, disable extensions etc.

**Reply by goodgame365** [reply]
Well, maybe it's my computer. I'll observe it for a while!
